client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx

```diff
@@ -157,6 +208,38 @@ function NewRideInfo() {
             }}
           />
         </FormControl>
+        <FormControl>
+          <TimePicker
+            sx={{ width: '100%' }}
+            label="שעת איסוף"
+            disablePast
+            ampm={false}
+            value={timeInIsrael}
+            onChange={(time) => {
+              if (time) {
```

Comment:
@aviv-taori What would happen to the time if the selected `DatePicker` will change?
I think this logic should be in a `useEffect` together with the logic of the `DatePicker onChange`.